.prettierignore

```diff
@@ -1 +1,2 @@
-resources/
\ No newline at end of file
+resources/
+*.snap
```

Comment:
Hmm, what happens without this? I thought Prettier didn't operate on snap files in the first place?

Reply:
I initially experienced a prettier failure that looked like it was caused by this, but actually that did not fix it. I removed this in 962df4e313e82526e937a111f987c975ba85da91